exercises/concept/squeaky-clean/SqueakyCleanTests.fs

```diff
@@ -0,0 +1,47 @@
+module SqueakyCleanTests
+
+open FsUnit.Xunit
+open Xunit
+open Exercism.Tests
+
+open SqueakyClean
+
+[<Fact>]
+[<Task(1)>]
+let ``Clean_empty_string``() = clean "" |> should equal ""
```

Comment:
I think that it would be useful for the student to have them implement the `let transform (c: char) : string` function first, before having to deal with the higher-order function. So we'd have the tests work on single characters first, and then later on have a couple of tests that verify that `clean` can work on an entire string.

This also has the benefit that it is more likely that students use the same exemplar solution, which gives great for mentoring.

Reply:
Agree. Update will follow later today.